extensions-core/parquet-extensions/src/test/java/org/apache/druid/data/input/parquet/WikiParquetReaderTest.java

```diff
@@ -77,7 +79,10 @@ public void testWiki() throws IOException
                                 + "  \"user\" : \"nuclear\",\n"
                                 + "  \"timestamp\" : \"2013-08-31T01:02:33Z\"\n"
                                 + "}";
-    Assert.assertEquals(expectedJson, DEFAULT_JSON_WRITER.writeValueAsString(sampled.get(0).getRawValues()));
+    ObjectMapper convertTreeTool = new ObjectMapper();
+    JsonNode expectedJson = convertTreeTool.readTree(expectedJsonString);
+    JsonNode targetJsonNode = convertTreeTool.readTree(DEFAULT_JSON_WRITER.writeValueAsString(sampled.get(0).getRawValues()));
+    Assert.assertEquals(expectedJson, targetJsonNode);
```

Comment:
Could you change this to a two-line change? Will give us a higher chance of acceptance by the devs:
```
ObjectMapper mapper = new ObjectMapper();
 Assert.assertEquals(mapper.readTree(expectedJson), mapper.readTree(DEFAULT_JSON_WRITER.writeValueAsString(sampled.get(0).getRawValues())));

```